### PR TITLE
Adds two watch-history articles 

### PR DESCRIPTION
**Pull request by lisad**
…as well as fixing the name of one destination in another article to be consistent

### CODE REVIEW

**Review by aaronjae22 — APPROVED** [review]

File: articles/watch-history1.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title:  Transferring your Instagram post/video viewing history elsewhere 
+datatype: "Viewing History"
+sources: Instagram
+destinations: ["Download myself", "Google Drive", "Dropbox"],
```
Comment (alexbainter) [inline articles/watch-history1.md:5]: Heads up @lisad, this trailing comma is apparently not valid syntax so the portmap deploy failed: https://github.com/dtinit/portmap/actions/runs/9587501977/job/26437681020#step:8:171

File: articles/watch-history2.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title:  Youtube video watching history  
+datatype: "Viewing History"
+sources: Youtube
+destinations: ["Download myself", "Google Drive", "Microsoft OneDrive",  "Dropbox", "Box"],
```
Comment (alexbainter) [inline articles/watch-history2.md:5]: Ditto

File: articles/watch-history3.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title:  Tiktok Video Browsing History  
+datatype: "Viewing History"
+sources: Tiktok
+destinations: ["Download myself"],
```
Comment (alexbainter) [inline articles/watch-history3.md:5]: :point_up:

**Review by alexbainter — COMMENTED** [review]
Made #68 to address!

**Pull request merged**